presentation/Main.java

```diff
@@ -0,0 +1,76 @@
+package presentation;
+
+import dao.Dao;
+import dao.StudentDao;
+import model.Student;
+import service.StudentService;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+
+    private static final int ADD_STUDENT=1;
```

Comment:
You can also move those constants for the menu items to your `Menu` class, make them `public`, and inside the `switch` cases, you can use `Menu.ADD_STUDENT`.